galvani/BioLogic.py

```diff
@@ -244,6 +245,9 @@ def VMPdata_dtype_from_colIDs(colIDs):
                 unique_field_name = field_name
             type_list.append((unique_field_name, field_type))
         else:
+            print('ColIDs: ',
+                  [colID for colID in colIDs if colID not in VMPdata_colID_flag_map])
+            print('Types: ', type_list[1:])
```

Comment:
I'm not sure how the types of the already-known columns are relevant here

Reply:
I  was using this in combination with a hex editor to find the type of the missing column but I found that the .mpt file has a different number of significant digits in the number whether it is 32 or 64. I will remove this print.